resource/translations/skosmos_sv.po

```diff
@@ -836,3 +836,12 @@ msgstr "Sökresultat"
 
 msgid "Sidebar listing: list and traverse vocabulary contents by a criterion"
 msgstr "Sidofältlista: ordna och bläddra begrepp efter kriterier"
+
+msgid "Subject:"
+msgstr "Ämne:"
+
+msgid "Write a subject"
+msgstr "Skriv ett ämne"
+
+msgid "Select a vocabulary"
+msgstr "Du kan välja ett vokabulär"
```

Comment:
I believe this should be "en vokabulär", not "ett vokabulär" (I checked the other Swedish translations)

Reply:
Corrected